criu/files-reg.c

```diff
@@ -1818,7 +1818,7 @@ int dump_one_reg_file(int lfd, u32 id, const struct fd_parms *p)
 	}
 
 	if (!skip_for_shell_job && mnt_is_overmounted(mi)) {
-		pr_err("Open files on overmounted mounts are not supported yet\n");
+		pr_err("Open files on overmounted mounts are not supported yet; %s fd=%d path=%s\n", ext_id, p->fd, link->name + 1);
```

Comment:
Can you please replace ext_id printing with just mnt_id printing? The variable ext_id sole purpose is for external file lookup, using it in other places does not look good.